src/templates/base.html

```diff
@@ -21,6 +21,7 @@
           gtag('js', new Date());
           gtag('config', 'UA-22381566-3');
       </script>
+      <script defer src="/static/js/bundle.js" nonce="{{ csp_nonce() }}"></script>
```

Comment:
Don’t need the nonce here - should only need it for inlined JS.

Can we have a better name than bundle.js? web-vitals.js?

I do wonder if we should move the GTM and GA config into one script with this (common.js?). Though the fact this is `defer` (deliberately I’m guessing to get the full load?) probably means that’s not a good idea.

Reply:
What I know about `defer` is that it reliably loads before `DOMContentLoaded`.
`async` may be loaded after.
I've added a listener on load and keeping `async`.